lib/koi.rb

```diff
@@ -1,26 +1,21 @@
 require 'compass-rails'
 require 'sass-rails'
 require 'csv'
-require 'devise'
```

Comment:
I moved it to the bottom of the file and commented it out, so it was grouped with the other gems that need re-enabling here once they've been updated and pushed.